### ISSUE TITLE
Multiple :require forms in ns macro confuse "organize imports"

### ISSUE THREAD

**Issue by NoahTheDuke** [clojure-lsp/clojure-lsp#1386]
**Describe the bug**

**To Reproduce**

Put the below in an empty file, run "organize imports" action on it. Only the first `:require` form is sorted and the `[b :refer [x]]` line removed. The second `:require` form is untouched.

Input:

```clojure
(ns example
  (:require
    [c]
    [b :refer [x]]
    [a])
  (:require
    [e]
    [d :refer [y]]
    [c]
    [b :refer [x]]
    [a]))
```

Output:

```clojure
(ns example
  (:require
   [a]
   [c])
  (:require
    [e]
    [d :refer [y]]
    [c]
    [b :refer [x]]
    [a]))
```

**Expected behavior**
All `:require` forms are organized.

**User details:**
 - OS: Ubuntu 22.04
 - Version: 2022.11.03-00.14.57

**Comment by ericdallo** [comment]
Agreed, TBH I didn't even know was possible to specify multiple :require/imports etc, is there any reason to one want that over a single one?

**Comment by ericdallo** [comment]
Oh got it, I think makes sense, should not be that hard to support that
